deepchem/models/tests/test_openvino.py

```diff
@@ -2,13 +2,55 @@
 import numpy as np
 import tensorflow as tf
 import torch
+import os
 
 import deepchem as dc
-from deepchem.molnet import load_tox21
+from deepchem.molnet import load_tox21, load_perovskite
 from deepchem.data import NumpyDataset
 from deepchem.models.torch_models import TorchModel
 
 
+def test_cgcnn_regression():
+  np.random.seed(123)
+  torch.manual_seed(124)
+
+  # load datasets
+  current_dir = os.path.dirname(os.path.abspath(__file__))
+  config = {
+      "reload": False,
+      "featurizer": dc.feat.CGCNNFeaturizer(),
+      # disable transformer
+      "transformers": [],
+      "data_dir": current_dir
+  }
+
+  tasks, datasets, transformers = load_perovskite(**config)
+  train, valid, _ = datasets
+  n_tasks = len(tasks)
+
+  def init(**kwargs):
+    torch.manual_seed(124)
+    return dc.models.CGCNNModel(
+        n_tasks=n_tasks,
+        mode='regression',
+        batch_size=1,
+        learning_rate=0.001,
+        **kwargs)
+
+  ref_model = init()
+  model = init(use_openvino=True)
+
+  # check overfit
+  metric = dc.metrics.Metric(dc.metrics.mae_score, n_tasks=n_tasks)
+  ref_scores = ref_model.evaluate(train, [metric], transformers)
+  scores = model.evaluate(train, [metric], transformers)
+
+  if os.path.exists(os.path.join(current_dir, 'perovskite.json')):
+    os.remove(os.path.join(current_dir, 'perovskite.json'))
+
+  assert abs(scores['mae_score'] - ref_scores['mae_score']) < 0.2
```

Comment:
Do we have other keys except `mae_score`?

Reply:
No

Reply:
So, maybe there is `predict` method? Because evaluate returns a metric, not predictions